pkg/acquisition/modules/docker/docker.go

```diff
@@ -409,6 +446,29 @@ func (d *DockerSource) EvalContainer(container dockerTypes.Container) (*Containe
 
 	}
 
+	if d.Config.UseContainerLabels {
+		parsedLabels := d.getContainerLabels(container.ID)
+		if len(parsedLabels) != 0 {
+			if v, ok := parsedLabels["enable"]; ok && strings.ToLower(v.(string)) == "true" {
```

Comment:
I think this can panic if the user creates a label like `crowdsec.enable.foo` as the type assertion will fail.

Reply:
I think its best for the program to panic 🤔 

Jokes aside, I will fix this should be as easy as checking the type cast before each use 🤷🏻